app/src/main/cpp/BrowserWorld.cpp

```diff
@@ -138,12 +142,14 @@ struct BrowserWorld::State {
   GroupPtr rootOpaqueParent;
   GroupPtr rootOpaque;
   GroupPtr rootTransparent;
+  GroupPtr rootNoBlitter;
```

Comment:
Please rename to rootController. I'll file a follow up to rename the others to rootEnv and rootUi. I don't think we actually need the rootOpaqueParent.